platform/linux-generic/odp_classification.c

```diff
@@ -191,7 +191,7 @@ int odp_cls_capability(odp_cls_capability_t *capability)
 	return 0;
 }
 
-void _odp_cls_update_hash_proto(cos_t *cos, odp_pktin_hash_proto_t hash_proto)
+static void _odp_cls_update_hash_proto(cos_t *cos, odp_pktin_hash_proto_t hash_proto)
```

Comment:
Checkpatch flags this line as being over 80 chars in len. Needs to be split to two lines.